cmd/config_volumes.go

```diff
@@ -0,0 +1,247 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/AlecAivazis/survey/v2/terminal"
+	"github.com/spf13/cobra"
+
+	bosonFunc "github.com/boson-project/func"
+	"github.com/boson-project/func/k8s"
+)
+
+func init() {
+	configCmd.AddCommand(configVolumesCmd)
+	configVolumesCmd.Flags().StringP("path", "p", cwd(), "Path to the project directory (Env: $FUNC_PATH)")
+	configVolumesCmd.AddCommand(configVolumesAddCmd)
+	configVolumesAddCmd.Flags().StringP("path", "p", cwd(), "Path to the project directory (Env: $FUNC_PATH)")
+	configVolumesCmd.AddCommand(configVolumesRemoveCmd)
+	configVolumesRemoveCmd.Flags().StringP("path", "p", cwd(), "Path to the project directory (Env: $FUNC_PATH)")
+}
+
+var configVolumesCmd = &cobra.Command{
+	Use:   "volumes",
+	Short: "List and manage configured volumes for a function",
+	Long: `List and manage configured volumes for a function
+
+Prints configured Volume mounts for a function project present in
+the current directory or from the directory specified with --path.
+`,
+	SuggestFor: []string{"volums", "volume", "vols"},
+	PreRunE:    bindEnv("path"),
+	RunE: func(cmd *cobra.Command, args []string) (err error) {
+		function, err := initConfigCommand(args)
+		if err != nil {
+			return
+		}
+
+		listVolumes(function)
+
+		return
+	},
+}
+
+var configVolumesAddCmd = &cobra.Command{
+	Use:   "add",
+	Short: "Add volume to the function configuration",
+	Long: `Add volume to the function configuration
+
+Interactive prompt that allows adding Secrets and ConfigMaps as Volume mounts to the function project
+in the current directory or from the directory specified with --path.
+`,
+	SuggestFor: []string{"ad", "create", "insert", "append"},
+	//ValidArgsFunction: CompleteFunctionList,
+	PreRunE: bindEnv("path"),
+	RunE: func(cmd *cobra.Command, args []string) (err error) {
+		function, err := initConfigCommand(args)
+		if err != nil {
+			return
+		}
+
+		return runAddVolumesPrompt(cmd.Context(), function)
+	},
+}
+
+var configVolumesRemoveCmd = &cobra.Command{
+	Use:   "remove",
+	Short: "Remove volume from the function configuration",
+	Long: `Remove volume from the function configuration
+
+	Interactive prompt that allows removing Volume mounts from the configuration of the function project
```

Comment:
```suggestion
	Interactive prompt to remove Volume mounts from the function project
```

`add` long description uses "as Volume mounts to the function project". But this says "removing Volume mounts from the configuration of the function project" which appears to have a different meaning and it's unclear if `add` would affect the cluster immediately or not. When we say "function project" we should always mean the configuration, so in this case I think using the words 'from the configuration" is probably redundant and could be confusing.